src/references/ethereum-units.json

```diff
@@ -15,7 +15,7 @@
   "basic_tx": 21000,
   "basic_transfer": 50000,
   "arbitrum_basic_tx": 735000,
-  "default_l1_gas_fee_optimism_swap": "1064760511747856",
+  "default_l1_gas_fee_optimism_swap": "0001064760511747",
```

Comment:
big where you getting this value from?

Reply:
i just figured since it was a 3 order of magnitude drop i'd add 3 zeros in front

Reply:
lmao im fine with this for now

Reply:
thats no bueno tho

Reply:
open to suggestions 🙏🏻